src/main/java/com/dapanda/jwt/JwtTokenProvider.java

```diff
@@ -132,4 +133,28 @@ public OAuthProvider getProviderFromToken(String token) {
 		return OAuthProvider.valueOf(raw);
 	}
 
+	public String resolveTokenFromCookie(HttpServletRequest request, String cookieName) {
+
+		if (request.getCookies() != null) {
+			for (Cookie cookie : request.getCookies()) {
+				if (cookieName.equals(cookie.getName())) {
+
+					return cookie.getValue();
+				}
+			}
+		}
+
+		return null;
```

Comment:
개인적으로는 빠른 return을 선호해서 이렇게 작성하면 depth도 줄고 가독성도 좋아질 것 같아요!
필요하다면 기존 방식 그대로 유지해도 괜찮습니다 :D 어떻게 생각하시나요?

```suggestion
		if (request.getCookies() == null) {
			
			return null;
		}
		
		for (Cookie cookie : request.getCookies()) {
			if (!cookieName.equals(cookie.getName())) {

				return null;
			}
			
			return cookie.getValue();
		}
```

Reply:
좋은데요~ 반영하겠습니다.